### DISCUSSION TITLE
Trouble with the Salary Prediction model

### DISCUSSION THREAD

**Discussion by SaiRuthvik12** [JoshitaReddy/TARP---Team-Up#3] [General]
Hey guys, I've been working on the Salary prediction model and I am having issues while trying to host it on Python Anywhere. The predict function of the model is not working and it isn't showing any error logs as well. Please let me know what I should do to fix this if you have faced a similar issue before, or is it a problem with Python Anywhere?

**Comment by Ananya322** [comment]
check the python code

**Comment by Ananya322** [comment]
Else I'll look into it and give an update regarding the same.

**Comment by JoshitaReddy** [comment]
https://www.pythonanywhere.com/forums/topic/28956/
https://stackoverflow.com/questions/64198347/python-anywhere-no-module-named-sklearn-linear-model-stochastic-gradient
Have a look at these links, they talk about debugging while hosting on PythonAnywhere
Are you facing similar errors?

**Reply by SaiRuthvik12** [reply]
I'll check these out. Thanks a lot
